### DISCUSSION TITLE
Trigger bunch based on focus mode

### DISCUSSION THREAD

**Discussion by xstrex** [ttscoff/bunch-docs#266] [Ideas]
From your documentation I can see that you can trigger a focus mode from a bunch, but it doesn't look like you can trigger a bunch from a focus mode. This would be really helpful for scheduled focus modes, such as work, or personal time. When work activates via it's schedule, trigger the work bunch, when work ends for the day, close the work bunch, and so on. 

I suspect this could probably be done with a shortcut, and maybe some duct tape and bubble gum, but a more elegant built-in method would be ideal.

**Comment by ttscoff** [comment]
I do not know how this would work, but apparently I haven't played much 
with Focus modes. I don't think they can trigger anything other than do 
not disturb settings, though?

You can always schedule Bunches using Bunch's built-in scheduling, and 
if you do want to go the shortcut route, it's easy to open, close, and 
toggle Bunches using the url handler or AppleScript library.
